engine/userguide/storagedriver/aufs-driver.md

```diff
@@ -1,251 +1,236 @@
-based on the `ubuntu:latest` image.
+## Configure Docker with the `aufs` storage driver
 
-![](images/aufs_layers.jpg)
+If the AUFS driver is loaded into the kernel when you start Docker, and no other
+storage driver is configured, Docker uses it by default.
 
-This diagram shows that each image layer, and the container layer, is
-represented in the Docker hosts filesystem as a directory under
-`/var/lib/docker/`. The union mount point provides the unified view of all
-layers. As of Docker 1.10, image layer IDs do not correspond to the names of
-the directories that contain their data.
+1.  Use the following command to verify that your kernel supports AUFS.
 
-AUFS also supports the copy-on-write technology (CoW). Not all storage drivers
-do.
+    ```bash
+    $ grep aufs /proc/filesystems
 
-## Container reads and writes with AUFS
+    nodev   aufs
+    ```
 
-Docker leverages AUFS CoW technology to enable image sharing and minimize the
-use of disk space. AUFS works at the file level. This means that all AUFS CoW
-operations copy entire files - even if only a small part of the file is being
-modified. This behavior can have a noticeable impact on container performance,
- especially if the files being copied are large, below a lot of image layers,
-or the CoW operation must search a deep directory tree.
+2.  Check which storage driver Docker is using.
 
-Consider, for example, an application running in a container needs to add a
-single new value to a large key-value store (file). If this is the first time
-the file is modified, it does not yet exist in the container's top writable
-layer. So, the CoW must *copy up* the file from the underlying image. The AUFS
-storage driver searches each image layer for the file. The search order is from
- top to bottom. When it is found, the entire file is *copied up* to the
-container's top writable layer. From there, it can be opened and modified.
+    ```bash
+    $ docker info
 
-Larger files obviously take longer to *copy up* than smaller files, and files
-that exist in lower image layers take longer to locate than those in higher
-layers. However, a *copy up* operation only occurs once per file on any given
-container. Subsequent reads and writes happen against the file's copy already
-*copied-up* to the container's top layer.
+    <truncated output>
+    Storage Driver: aufs
+     Root Dir: /var/lib/docker/aufs
+     Backing Filesystem: extfs
+     Dirs: 0
+     Dirperm1 Supported: true
+    <truncated output>
+    ```
 
-## Deleting files with the AUFS storage driver
+3.  If you are using a different storage driver, then it means that either AUFS
+    is not included in the kernel (in which case a different default driver will
+    be used) or that means that Docker has been explicitly configured to use a
+    different driver. Check `/etc/docker/daemon.json` or the output of
+    `ps auxw |grep dockerd` to see if Docker has been started with the
```

Comment:
That really doesn't matter to the shell, but I'll add the space because it looks prettier.